src/integrationTest/resources/application.yaml

```diff
@@ -167,6 +168,7 @@ docmosis:
       FINAL_ORDER_GRANTED_COVER_LETTER: FL-NFD-GOR-ENG-Final-Order-Cover-Letter.docx #TODO: add translated content
       CO_GRANTED_COVER_LETTER: FL-NFD-GOR-ENG-Conditional-Order-Granted-Cover-Letter.docx #TODO: update welsh content
       SWITCH_TO_SOLE_CO_LETTER: FL-NFD-GOR-ENG-Applied-For-Co-Switch-To-Sole.docx #TODO: add translated content
+      NFD_NOP_APP1_JS_SOLE: FL-NFD-GOR-WEL-Notice_Of_Proceedings_Applicant1_JS_Sole.docx
```

Comment:
best to check with Nitin and Nicola on these welsh changes as the tickets for all welsh JS changes have been withdrawn